docs/resources/templates/LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
-
-Copyright (c) 2017 Microsoft Corporation
-
```

Comment:
docs/source-control/README.md is using this file as a link, might be a broken ref?